src/fides/api/schemas/connection_configuration/connection_type_system_map.py

```diff
@@ -20,3 +20,5 @@ class ConnectionSystemTypeMap(BaseModel):
     user_guide: Optional[str] = None
     supported_actions: List[ActionType]
     model_config = ConfigDict(use_enum_values=True, from_attributes=True)
+    is_custom: bool = False
+    file_connector_available: bool = False
```

Comment:
This is also user-facing so maybe something like this? And because this is a Pydantic model, you can add descriptions that will show up in the Swagger docs. I know we don't usually do this but let's start asking for this moving forward.
```suggestion
    default_connector_available: bool = Field(
        default=False,
        description="Indicates whether a Fides-provided default connector template is available",
    )
```

Reply:
Makes it a bit nicer
![image](https://github.com/user-attachments/assets/72e338ab-3d98-4bcf-8127-97a88cc87c58)